exodus_core/analysis/static_analysis.py

```diff
@@ -509,22 +509,22 @@ def print_apk_infos(self):
         libraries = self.get_libraries()
         certificates = self.get_certificates()
         print("=== Information")
-        print('- APK path: %s' % self.apk_path)
-        print('- APK sum: %s' % self.get_sha256())
-        print('- App version: %s' % self.get_version())
-        print('- App version code: %s' % self.get_version_code())
-        print('- App UID: %s' % self.get_application_universal_id())
-        print('- App name: %s' % self.get_app_name())
-        print('- App package: %s' % self.get_package())
-        print('- App permissions: %s' % len(permissions))
+        print('- APK path: {0}'.format(self.apk_path))
```

Comment:
I think we should replace all `{0}` by `{}`, as the 0 is not useful here.

Reply:
Good idea. I use to believe I have to add a number in the brackets necessary due to an old example.